internal/tools/go.mod

```diff
@@ -233,15 +250,3 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20230312165513-e84e2d14e3b8 // indirect
 )
-
```

Comment:
Can we keep this here instead of moving to the top?

Reply:
yup! i can revert that auto formatting change.

Reply:
done ✅